src/components/EncounterNotes/SlothasorNotes.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+
+import RoleEmbed from "../RoleEmbed/RoleEmbed";
+import GW2Skill from "../GW2Skill/GW2Skill";
+import BoonEmbed from "../BoonEmbed/BoonEmbed";
+
+function SlothasorNotes() {
+  return (
+    <div>
+      <h3>General</h3>
+      <ul>
+        <li>Keep Sloth at beginning until CC at 80%.</li>
+        <li>
+          <RoleEmbed role="Heal" profession="firebrand" /> provides
+          <BoonEmbed id="stability" /> after each CC phase.
+        </li>
+        <li>Tank as close to the middle as possible (face towards middle)</li>
+        <li>
```

Comment:
I think nowadays we aren't even running two chrono's - or at least aren't relying on them. So we should probably emphasise that the chrono and firebrand **need** to work together to pull slublings.

Should probably remove Chrono from the composition tbh.

Reply:
True, but I'd still wait till we have proper 10 men static. Because for know there's still a chance that new people will main chronos